lib/membership/member.js

```diff
@@ -170,10 +170,13 @@ Member.prototype._applyUpdatePenalty = function _applyUpdatePenalty() {
     }
 };
 
-Member.prototype._isLocalOverride = function _isLocalOverride(update) {
-    var self = this;
+// Imagine I am ringpop, and I get an update. If the update is about me, then
+// it can be a localOverride. If it is a localOverride, then the update will be
+// applied. If it's not, then the update will be skipped.
```

Comment:
This comment isn't completely correct. If `_isLocalOverride` returns true, the update will be changed to assert aliveness (and bump incarnationNumber); if not and it's also not an other overwrite, it's skipped.